packages/bot/src/assets/sass/saveload-modal.scss

```diff
@@ -55,18 +55,39 @@
 
 .saveload-type {
     &__checkbox-text {
-        @include typeface(--title-left-bold-black, capitalize);
+        font-size: 14px;
+        font-weight: bold;
+        font-style: normal;
+        font-stretch: normal;
```

Comment:
These are redundant if their parent hasn't changed the default value. Can be removed.